tensorflow_lite_support/cc/test/testdata/task/vision/BUILD

```diff
@@ -0,0 +1,19 @@
+package(
+    default_visibility = ["//tensorflow_lite_support:users"],
+    licenses = ["notice"],  # Apache 2.0
+)
+
+filegroup(
+    name = "test_models",
+    srcs = glob([
+        "*.tflite",
+    ]),
+)
+
+filegroup(
+    name = "test_images",
```

Comment:
Just curious what's the license for this files?

Reply:
The models are all open sourced. But the images, I'm not sure. Some of them are used in our C++ demo examples for a while, so I assumed them should be OK to be open sourced. But let me verify it with the VisionKit team.